### ISSUE TITLE
Workbook written by pylightxl crashes Excel

### ISSUE THREAD

**Issue by luphord** [PydPiper/pylightxl#50]
**Pylightxl Version**: 1.55
**Python Version**: 3.8

**Summary of Bug/Feature**:
Opening an Excel workbook written by `pylightxl`, then opening the same workbook again (without closing it first) crashes Excel. Also, it is not possible to add new Worksheets to these workbooks. Clicking `+` fails with the error message `This action won't work on multiple selections.`.

![error_message](https://user-images.githubusercontent.com/46622741/127524730-8eb7e0c2-8db3-4056-9b4e-fc5637101e09.png)

For generating the workbook I did nothing special, just used this example from the docs:

```python
import pylightxl as xl

# take this list for example as our input data that we want to put in column A
mydata = [10,20,30,40]

# create a black db
db = xl.Database()

# add a blank worksheet to the db
db.add_ws(ws="Sheet1")

# loop to add our data to the worksheet
for row_id, data in enumerate(mydata, start=1):
    db.ws(ws="Sheet1").update_index(row=row_id, col=1, val=data)

# write out the db
xl.writexl(db=db, fn="output.xlsx")
```

**Traceback**:
N/A (no exception)

**Suggestion for fix**:
The issue seems to be caused by the `<sheetViews>...</sheetViews>` element written to a worksheet. Removing it as in the screenshot below results in working worksheets for me. A [GitHub issue on box/spout](https://github.com/box/spout/issues/818) pointed me to this. Do the sheetViews serve any purpose or could they just be removed?

![Screenshot from 2021-07-29 17-57-41](https://user-images.githubusercontent.com/46622741/127525359-40f32496-31cf-4599-8088-171c4aabee9c.png)

**Comment by PydPiper** [comment]
Hi @luphord thank you for considering using pylightxl for your project. I am taking a look at the issue now

**Comment by PydPiper** [comment]
Thank you for bring this to our attention, you are correct this is in fact a bug around sheetViews. sheetViews is the the pointer for excel to reopen a workbook in the last saved worksheet tab view. This is an extra feature that is unnecessary for pylightxl and therefore i removed it. This change will be reflected in v1.56

**Comment by luphord** [comment]
Awesome, thanks for the quick response @PydPiper!

I can confirm that the issue is fixed in v1.56.
